modular_skyrat/modules/emote_panel/code/emote_panel.dm

```diff
@@ -0,0 +1,978 @@
+		///mob/living/carbon/human/proc/emote_hug,
+		/mob/living/carbon/human/proc/emote_mumble,
+		/mob/living/carbon/human/proc/emote_pale,
+		/mob/living/carbon/human/proc/emote_raise,
+		/mob/living/carbon/human/proc/emote_salute,
+		/mob/living/carbon/human/proc/emote_shrug,
+		/mob/living/carbon/human/proc/emote_wag,
+		/mob/living/carbon/human/proc/emote_wing
+	)
+	all_emotes += human_emotes
+	if(istype(src, /mob/living/carbon/human))
+		available_emotes += human_emotes
+
+	// modular_skyrat\modules\emotes\code\emote.dm
+	var/list/skyrat_living_emotes = list(
+		/mob/living/proc/emote_peep,
+		/mob/living/proc/emote_peep2,
+		/mob/living/proc/emote_snap,
+		/mob/living/proc/emote_snap2,
+		/mob/living/proc/emote_snap3,
+		/mob/living/proc/emote_awoo,
+		/mob/living/proc/emote_nya,
+		/mob/living/proc/emote_weh,
+		/mob/living/proc/emote_mothsqueak,
+		/mob/living/proc/emote_mousesqueak,
+		/mob/living/proc/emote_merp,
+		/mob/living/proc/emote_bark,
+		/mob/living/proc/emote_squish,
+		/mob/living/proc/emote_meow,
+		/mob/living/proc/emote_hiss1,
+		/mob/living/proc/emote_chitter,
+		/mob/living/proc/emote_snore,
+		/mob/living/proc/emote_clap,
+		/mob/living/proc/emote_clap1,
+		/mob/living/proc/emote_headtilt,
+		/mob/living/proc/emote_blink2,
+		/mob/living/proc/emote_rblink,
+		/mob/living/proc/emote_squint,
+		/mob/living/proc/emote_smirk,
+		/mob/living/proc/emote_eyeroll,
+		/mob/living/proc/emote_huff,
+		/mob/living/proc/emote_etwitch,
+		/mob/living/proc/emote_clear,
+		/mob/living/proc/emote_bawk,
+		/mob/living/proc/emote_caw,
+		/mob/living/proc/emote_caw2,
+		/mob/living/proc/emote_whistle,
+		/mob/living/proc/emote_blep,
+		/mob/living/proc/emote_bork,
+		/mob/living/proc/emote_hoot,
+		/mob/living/proc/emote_growl,
+		/mob/living/proc/emote_woof,
+		/mob/living/proc/emote_baa,
+		/mob/living/proc/emote_baa2,
+		/mob/living/proc/emote_wurble,
+		/mob/living/proc/emote_rattle,
+		/mob/living/proc/emote_cackle,
+		/mob/living/proc/emote_warble,
+		/mob/living/proc/emote_trills,
+		/mob/living/proc/emote_rpurr,
+		/mob/living/proc/emote_purr,
+		/mob/living/proc/emote_moo,
+		/mob/living/proc/emote_honk1
+	)
+	all_emotes += skyrat_living_emotes
+	if(istype(src, /mob/living))
+		available_emotes += skyrat_living_emotes
+
+	// code\modules\mob\living\brain\emote.dm
+	var/list/brain_emotes = list(
+		/mob/living/brain/proc/emote_alarm,
+		/mob/living/brain/proc/emote_alert,
+		/mob/living/brain/proc/emote_flash,
+		/mob/living/brain/proc/emote_notice,
+		/mob/living/brain/proc/emote_whistle_brain
+	)
+	all_emotes += brain_emotes
+	if(istype(src, /mob/living/brain))
+		available_emotes += brain_emotes
+
+	// code\modules\mob\living\carbon\alien\emote.dm
+	var/list/alien_emotes = list(
+		/mob/living/carbon/alien/proc/emote_gnarl,
+		/mob/living/carbon/alien/proc/emote_hiss,
+		/mob/living/carbon/alien/proc/emote_roar
+	)
+	all_emotes += alien_emotes
+	if(istype(src, /mob/living/carbon/alien))
```

Comment:
isalien

Reply:
Appropriate check applied, also moved the checks to make it easier to read